.vscode/settings.json

```diff
@@ -1,4 +1,5 @@
 {
   "python.formatting.provider": "black",
-  "python.analysis.extraPaths": ["${workspaceFolder}/oasst-shared"]
+  "python.analysis.extraPaths": ["${workspaceFolder}/oasst-shared"],
+  "prettier.singleQuote": false
```

Comment:
isn't this the default value?

Reply:
Yes but my prettier config in the local is different. Without this change prettier will use single quote.